src/backend/gifs.nim

```diff
@@ -0,0 +1,13 @@
+import json, strutils
+import core, utils
+import response_type
+
+export response_type
+
+proc getRecentGifs*(): RpcResponse[JsonNode] {.raises: [Exception].} =
+  let payload = %* []
+  result = callServicePrivateRPC("getRecentGifs", payload, "gif")
```

Comment:
WDYT to still keep only one endpoint to call RPC (`callPrivateRPC`) and instead do the following:
`callPrivateRPC(serviceMethod("gif", "getRecentGifs"), payload)` ?

Reply:
I think, we might as well also just do `callPrivateRPC("gif_getRecentGifs", payload)`